### PR TITLE
docs(css): don't add outer styling to base reset class

### PR DESCRIPTION
**Pull request by alexkrolick**

### CODE REVIEW

**Comment by boxcla** [comment]
Verified that @alexkrolick has signed the CLA. Thanks for the pull request!

File: src/elements/content-preview/README.md [hunk 1]
```diff
@@ -23,6 +23,7 @@ var ContentPreview = require('./ContentPreview').default;
         features={FEATURES}
         fileId={FILE_ID}
         token={TOKEN}
+        className="example"
```
Comment (jstoffan) [inline src/elements/content-preview/README.md]: Can we avoid adding an additional prop here by targeting each Element's root class name directly?

```
.be {
    &.bce,
    &.bcs,
    &.bcow,
    &.bcp,
    &.bcpr,
    &.bcu {
        box-shadow: 0 0 20px 0 rgba(85, 85, 85, .42);
    }
}

Comment (jstoffan) [reply src/elements/content-preview/README.md]: The current change will result in box shadows being lost from all other examples, so a solution similar to the above will be necessary for us to merge.

**Pull request merged**